packages/terra-date-time-picker/src/_TimeClarification.module.scss

```diff
@@ -1,29 +1,29 @@
 /* stylelint-disable selector-max-compound-selectors, max-nesting-depth */
 :local {
   .time-clarification {
-    background-color: var(--terra-date-time-clarification-modal-background-color, #fff);
-    border: var(--terra-date-time-clarification-modal-border);
-    border-radius: var(--terra-date-time-clarification-modal-border-radius, 5px);
-    box-shadow: var(--terra-date-time-clarification-modal-box-shadow, 0 3px 7px rgba(0, 0, 0, 0.3));
-    color: var(--terra-date-time-clarification-modal-foreground-color);
+    background-color: var(--terra-date-time-picker-clarification-modal-background-color, #fff);
+    border: var(--terra-date-time-picker-clarification-modal-border);
+    border-radius: var(--terra-date-time-picker-clarification-modal-border-radius, 5px);
+    box-shadow: var(--terra-date-time-picker-clarification-modal-box-shadow, 0 3px 7px rgba(0, 0, 0, 0.3));
+    color: var(--terra-date-time-picker-clarification-modal-foreground-color);
```

Comment:
We can drop the word `foreground` from this variable

Reply:
https://github.com/cerner/terra-core/pull/2125/commits/e4b290594ca88b9490fc043c617853b7e1f533ff